.devcontainer/docker-compose.yaml

```diff
@@ -60,6 +60,13 @@ services:
         source: ../app
         target: /app
 
+  caddy:
+    image: caddy
+    volumes:
+      - "../caddy/Caddyfile.dev:/etc/caddy/Caddyfile"
+    ports:
+      - "3000:3000"
```

Comment:
Can we not use 80 because it's a port < 1024 hence privileged on Linux?

We may also remove the lines above in the strapi service if the reverse proxy works correctly. Or do you think we should keep it for debugging purposes? In which case, consider running caddy on port 8000 for consistency.
```
      # Expose strapi to port 8001
      - "8001:8001"
```